webpack/dev.config.js

```diff
@@ -7,6 +7,12 @@ const port = 3000
 const customPath = path.join(__dirname, './customPublicPath')
 const hotScript = 'webpack-hot-middleware/client?path=__webpack_hmr&dynamicPublicPath=true'
 
+let buildDir = process.env.TARGET_GEN_DIR
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
as noted above, we should just set `process.env.TARGET_GEN_DIR` to the correct value to start with